spyder_kernels/console/kernel.py

```diff
@@ -40,23 +40,78 @@
 class SpyderShell(ZMQInteractiveShell):
```

Comment:
Let's move this class to another file called  `shell.py` (next to this one) because now it's too big to be here.